src/components/App/App.css

```diff
@@ -1,10 +1,27 @@
+html {
+  background-color: #4B658B;
+  height: 100%;
+  font-family: 'Comfortaa';
+}
+
```

Comment:
Here, HTML rules are applied to ensure there is no white space when components are re-sized.